### DISCUSSION TITLE
Download of maps shows counter (1) even if there is no update

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#197] [Q&A] [answered]
Download of maps shows counter (1) even if there is no update

**Comment by devemux86** [comment]
Which screen of the 3 do you mean?
The first 2 screens show all updates.

<img src="https://github.com/devemux86/cruiser/assets/3484020/e44d6269-9b23-4b3a-8e82-3a73dc760597" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/c24d0514-9d33-4d0a-aec4-8d44830f8f79" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/c041262a-7784-40fb-8193-0a84ee4efcd2" width="33%"/>

**Reply by DolomitesTom** [reply]
the first one, the other is correct

**Reply by devemux86** [reply]
> the other is correct

The 2nd or the 3rd?

How is your 3rd screen?
Do you have updates in POI or GraphHopper?

**Reply by Ciutnik** [reply]
The counter also shows a number when there is a theme update.

**Comment by devemux86** [comment, marked as answer]
@DolomitesTom thanks for the report.

Because of the need of Map + POI updates (see https://github.com/devemux86/cruiser/discussions/189#discussioncomment-6062373),
next Cruiser will have an improved update implementation, so the counter should behave better. :slightly_smiling_face:

**Reply by DolomitesTom** [reply]
Better now thanks
